python/rapids/worker.py

```diff
@@ -52,6 +52,8 @@ def initialize_gpu_mem():
                              "`RAPIDS_POOLED_MEM_SIZE`.")
         if pool_max_size == 0:
             pool_max_size = max_size
+        pool_max_size = pool_max_size >> 8 << 8
+        pool_size = pool_size >> 8 << 8
```

Comment:
What is this change doing in this PR?

Reply:
Ah, I now see it was explained in the description as a bug found during the update to 0.16.